bigtop-packages/src/common/ignite-hadoop/do-component-build

```diff
@@ -21,4 +21,39 @@ set -ex
 export MAVEN_OPTS="-Xmx512M"
 
 sed -i '/asm-all/{n;s/4.2/5.0.2/}' modules/hadoop/pom.xml
+
+if [ "$HOSTTYPE" != "x86_64" ] ; then
+    SAVED_MD5="8b92df116216298ea76a5165ed90ac1f"
+    SAVED_SHA1="ecfa2478af45e2b4059a3a97686308041d220031"
+    # download ignite-shmem from maven central
+    mvn dependency:get -DremoteRepositories=https://repo1.maven.org/maven2/ -DgroupId=org.gridgain -DartifactId=ignite-shmem -Dversion=1.0.0
+    mkdir tmp-build
+    cd tmp-build
+    # download source jar
+    wget https://repo1.maven.org/maven2/org/gridgain/ignite-shmem/1.0.0/ignite-shmem-1.0.0-sources.jar
+    # verify it
+    md5sum=$(/usr/bin/md5sum ignite-shmem-1.0.0-sources.jar | cut -d ' ' -f 1)
+    sha1sum=$(/usr/bin/sha1sum ignite-shmem-1.0.0-sources.jar | cut -d ' ' -f 1)
+    if [ $md5sum != $SAVED_MD5 ] || [ $sha1sum != $SAVED_SHA1 ]; then
+        echo "Downloaded file checksum doesn't match!"
```

Comment:
I'd prefer to have ignite-shmem-1.0.0-sources.jar in the error message so that it's super clear what's going wrong :)

Reply:
Got it and updated it.